docs/doc_assets/js/components/sticky-nav.js

```diff
@@ -7,20 +7,22 @@ var STICKY_CLASS_NAME = 'is-scrolled';
 module.exports = function stickyNav (event) {
 
   var originalNavigationHeight = $nav.outerHeight(true);
-  var bannerHeight = $('.usa-site-header').outerHeight(true) - originalNavigationHeight * 0.5;
+  var bannerHeight = $('.usa-site-header').outerHeight(true) - originalNavigationHeight * 0.5 - 50;
```

Comment:
@rogeruiz I updated the scroll location to pick it up at the right spot, but it doesn't scroll back up properly.